<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="kafka.integration.FetcherTest" tests="1" skipped="0" failures="0" errors="0" timestamp="2015-03-27T21:41:00" hostname="airbears2-10-142-35-31.AirBears2.1918.Berkeley.EDU" time="1.451">
  <properties/>
  <testcase name="testFetcher" classname="kafka.integration.FetcherTest" time="1.451"/>
  <system-out><![CDATA[[2015-03-27 14:41:00,840] ERROR Closing socket for /127.0.0.1 because of error (kafka.network.Processor:103)
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:197)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:379)
	at kafka.utils.Utils$.read(Utils.scala:380)
	at kafka.network.BoundedByteBufferReceive.readFrom(BoundedByteBufferReceive.scala:54)
	at kafka.network.Processor.read(SocketServer.scala:444)
	at kafka.network.Processor.run(SocketServer.scala:340)
	at java.lang.Thread.run(Thread.java:744)
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
